Task 16.1.py

```diff
@@ -0,0 +1,52 @@
+class Person:
+    def __init__(self, *args, **kwargs):

```

Comment:
Since every person has name and surname and age I would do `(self, name, surname, age)` and would remove these three parameters from the children classes. I will show how to do it below.